tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,36}
+envlist = py{27,35}
```

Comment:
Need to add `py36` here for the updated `validate` make target to work.

Reply:
I think tox will run py36 without it if explicitly asked, but yes, we want it there.  Fixed.